src/hooks/useOnAvatarPress.ts

```diff
@@ -94,22 +100,34 @@ export default () => {
       setNextEmoji();
       return;
     }
-    const avatarActionSheetOptions = [
-      'Choose from Library',
-      ...(!accountImage ? ['Pick an Emoji'] : []),
-      ...(accountImage ? ['Remove Photo'] : []),
-      ...(ios ? ['Cancel'] : []),
-    ];
 
-    showActionSheetWithOptions(
-      {
-        cancelButtonIndex: avatarActionSheetOptions.length - 1,
-        destructiveButtonIndex: accountImage
-          ? avatarActionSheetOptions.length - 2
-          : undefined,
-        options: avatarActionSheetOptions,
-      },
-      async (buttonIndex: Number) => {
+    const isENSProfile =
+      profilesEnabled &&
+      ensProfile?.data?.owner?.address?.toLowerCase() ===
```

Comment:
Wonder if we should have an `isOwner` attribute within `useENSProfile`, that matches on the account address? Would make abstraction in other places easier too.

Reply:
right! forgot it again